sklearn/linear_model/coordinate_descent.py

```diff
@@ -132,17 +132,22 @@ def lasso_path(X, y, eps=1e-3, n_alphas=100, alphas=None,
 
     The Lasso optimization function varies for mono and multi-outputs.
 
-    For mono-output tasks it is::
+    For mono-output tasks it is
 
-        (1 / (2 * n_samples)) * ||y - Xw||^2_2 + alpha * ||w||_1
+    .. math::
+        \\frac{1}{2 * \\mathrm{n\\_samples}} * \\|y - Xw\\|^2_2
```

Comment:
Do not use `*` for multiplications this looks weird. Either do not use anything or in the few cases where it helps readability use `\times`.